cdc/processor/pipeline/sink.go

```diff
@@ -315,17 +322,47 @@ func (n *sinkNode) Receive(ctx pipeline.NodeContext) error {
 			failpoint.Inject("ProcessorSyncResolvedError", func() {
 				failpoint.Return(errors.New("processor sync resolved injected error"))
 			})
-			if err := n.flushSink(ctx, msg.PolymorphicEvent.CRTs); err != nil {
+
+			err := retry.Do(ctx, func() error {
+				err := n.flushSink(ctx, msg.PolymorphicEvent.CRTs)
+				if err != nil {
+					return err
+				}
+				return nil
+			}, retry.WithBackoffBaseDelay(20),
+				retry.WithInfiniteTries(),
```

Comment:
Please comment on when `flushSink` will be retried. Also can you print a warning if retry keeps failing after a certain duration (say 10 seconds)?

Reply:
got